includes/apple-exporter/builders/class-component-layouts.php

```diff
@@ -105,11 +109,11 @@ public function set_anchor_layout_for( $component ) {
 		if ( ! $this->layout_exists( $layout_name ) ) {
 
 			// Cache settings.
-			$body_orientation = $this->get_setting( 'body_orientation' );
-			$body_offset = $this->get_setting( 'body_offset' );
-			$alignment_offset = $this->get_setting( 'alignment_offset' );
-			$body_column_span = $this->get_setting( 'body_column_span' );
-			$layout_columns = $this->get_setting( 'layout_columns' );
+			$body_orientation = $theme->get_value( 'body_orientation' );
+			$body_offset = $theme->get_body_offset();
```

Comment:
They were named functions previously, but there was some logic in the `Settings` class that checked for the existence of the function before checking for the existence of a key/value pair in the internal settings array. Since I ended up moving this functionality to the theme class anyway, I made the reference transparent, since it's a computed value rather than one that is actually stored in the settings option.